packages/next/src/server/app-render/validate-url.tsx

```diff
@@ -1,10 +1,25 @@
 const DUMMY_ORIGIN = 'http://n'
 const INVALID_URL_MESSAGE = 'Invalid request URL'
 
+// URL.canParse is supported on Node 19 and 20.
+// Node 18 backport is pending.
+// Reference: https://github.com/nodejs/node/pull/48345
+const supportsCanParse = URL.hasOwnProperty('canParse')
+
 export function validateURL(url: string | undefined): string {
-  if (!url) {
+  if (url == null) {
```

Comment:
Should this be `== undefined`?

Reply:
`== null` checks for both `undefined` and `null`. This is pretty common in Node.js repository: https://github.com/search?q=repo%3Anodejs%2Fnode+%22+%3D%3D+null%22+path%3Alib&type=code